compliance/b_10.rst

```diff
@@ -0,0 +1,241 @@
+.. _compliance_b_10:
+
+====================================================
+170.315 (b)(10) Electronic Health Information Export
+====================================================
+
+`170.315 (b)(10) requires the implementation of Electronic Health Information (EHI) export functionality <https://www.healthit.gov/test-method/electronic-health-information-export>`_. 
+This encompasses the ability to generate export files for individual patients and entire patient populations in a computable format. 
+Firely Server provides comprehensive support for this requirement through its :ref:`Bulk Data Export feature <feature_bulkdataexport>`.
+For a deeper understanding and comprehensive utilization of this feature beyond the (b)(10) requirements, 
+please consult the provided documentation, which includes setup instructions, application settings, parameters, filtering options, and additional details.
+
+File Format
+===========
+
+Bulk Data Export files are formatted in NDJSON (New line delimited JSON), a modified version of the JSON format designed for efficient bulk data transfer. 
+NDJSON follows a simplified JSON structure where resources are serialized without whitespace and separated by a newline pair (characters 13 and 10). 
+This format aids in streamlining processing by ensuring that each NDJSON document contains only resources of a single type. 
+Each line within the document represents a resource of that specific type. 
+It is worth noting that while resources can include contained resources of various types, a single file can accommodate multiple resources for multiple patients with multiple records.
```

Comment:
I guess this sentence confuses me a bit: 
An NDJSON document can only contain resources of one type (such as Patient, Observation etc.), the latter sentence of line 20  however almost seems to suggest that a single NDJSON file can contain multiple _different_ resource types. Could you clarify?

Reply:
I'm stuck on this. Do you have a suggestion? In context of the other sentences it seems clear to me.